### ISSUE TITLE
Quota does not update after login

### ISSUE THREAD

**Issue by zeigerpuppy** [RainLoop/rainloop-webmail#1214]
The quota shown in rainloop for the logged-in email account does not update during the session.
This can result in the quota reporting an incorrect value if new emails are received or deleted.

- refreshing the quota on emptying of trash would be especially useful to allow users with full accounts to see the results of their actions.

**Comment by Darlelet** [comment]
Yeah you're right, seems pretty useful to have the quota refreshed when dumping large mailboxes, or deleting a significant amount of emails.
